topdown/http.go

```diff
@@ -110,6 +114,15 @@ func executeHTTPRequest(bctx BuiltinContext, obj ast.Object) (ast.Value, error)
 		}
 	}
 
+	// check if redirects are enabled
+	if enableRedirect {
+		client.CheckRedirect = nil
+	} else {
+		client.CheckRedirect = func(req *http.Request, via []*http.Request) error {
```

Comment:
[nit] We could go with ` func(*http.Request, []*http.Request) error` since we're not using the arguments